src/Components/Header/Nav.jsx

```diff
@@ -61,7 +61,7 @@ export const Nav = () => {
                     <Link className="flex flex-column" to="/wishlist">
                         <span className="container-badge">
                             <span className="material-icons text-2xl nav-links nav-wishlist">favorite</span>
-                            <span className="status-badge status-badge-number">2</span>
+                            <span className={`${wishlist.length>0 ? "status-badge status-badge-number" : "display-none"}`}>{wishlist.length}</span>
```

Comment:
You can take advantage of React here. You can achieve this without class toggling

`{wishlist.length>0 && <span className="status-badge status-badge-number">{wishlist.length}</span>}`

Reply:
yes got it, will change this. Thank you for suggesting this.